tests/flow/test_data/redis_scripts.py

```diff
@@ -0,0 +1,66 @@
+
+def redis_string_int_to_tensor(redis_value: Any):
+    return torch.tensor(int(str(redis_value)))
```

Comment:
One thing we need to make sure is that we move the tensor to the right device. We can ask the user to do so, but it needs to be done otherwise a script running on GPU will find itself with an input on CPU and will fail.

Reply:
I would just add a note here (and maybe in the docs) mentioning the device placement.
The ideal solution here would be to have a `SCRIPT_DEVICE` global to allow a user to do
```
a_tensor = torch.tensor(int(str(redis_value))).to(device=SCRIPT_DEVICE)
```
and have tensors created within the script be located on the same device as the eventual script inputs.